guides/keras_nlp/getting_started.py

```diff
@@ -0,0 +1,619 @@
+        "mask_positions": masked_inputs["mask_positions"],
+    }
+    labels = masked_inputs["mask_ids"]
+    weights = masked_inputs["mask_weights"]
+    return features, labels, weights
+
+
+pretrain_ds = imdb_train.map(preprocess, num_parallel_calls=tf.data.AUTOTUNE).prefetch(
+    tf.data.AUTOTUNE
+)
+pretrain_val_ds = imdb_test.map(
+    preprocess, num_parallel_calls=tf.data.AUTOTUNE
+).prefetch(tf.data.AUTOTUNE)
+
+# Tokens with ID 103 are "masked"
+print(pretrain_ds.unbatch().take(1).get_single_element())
+
+"""
+### Pretraining model
+"""
+
+# BERT backbone
+backbone = keras_nlp.models.BertBackbone(
+    vocabulary_size=tokenizer.vocabulary_size(),
+    num_layers=2,
+    num_heads=2,
+    hidden_dim=128,
+    intermediate_dim=512,
+)
+
+# Language modeling head
+mlm_head = keras_nlp.layers.MaskedLMHead(
+    embedding_weights=backbone.token_embedding.embeddings,
+)
+
+inputs = {
+    "token_ids": keras.Input(shape=(None,), dtype=tf.int32),
+    "segment_ids": keras.Input(shape=(None,), dtype=tf.int32),
+    "padding_mask": keras.Input(shape=(None,), dtype=tf.int32),
+    "mask_positions": keras.Input(shape=(None,), dtype=tf.int32),
+}
+
+# Encoded token sequence
+sequence = backbone(inputs)["sequence_output"]
+
+# Predict an output word for each masked input token.
+# We use the input token embedding to project from our encoded vectors to
+# vocabulary logits, which has been shown to improve training efficiency.
+outputs = mlm_head(sequence, mask_positions=inputs["mask_positions"])
+
+# Define and compile our pretraining model.
+pretraining_model = keras.Model(inputs, outputs)
+pretraining_model.summary()
+pretraining_model.compile(
+    loss=keras.losses.SparseCategoricalCrossentropy(from_logits=True),
+    optimizer=keras.optimizers.experimental.AdamW(learning_rate=5e-4),
+    weighted_metrics=keras.metrics.SparseCategoricalAccuracy(),
+    jit_compile=True,
+)
+
+# Pretrain on IMDB dataset
+pretraining_model.fit(
+    pretrain_ds,
+    validation_data=pretrain_val_ds,
+    epochs=3,  # Increase to 6 for higher accuracy
+)
+
+"""
+After pretraining save your `backbone` submodel to use in a new task!
+"""
+
+"""
+## Build and train your own transformer from scratch
+![picture](https://drive.google.com/uc?id=1pzwLPCtvzmHY3DKzH-MBzmjWFJ3pKVB5)
+
+Want to implement a novel transformer architecture? The KerasNLP library offers all the
+low-level modules used to build SoTA architectures in our `models` API. This includes the
+`keras_nlp.tokenizers` API which allows you to train your own subword tokenizer using
+`WordPieceTokenizer`, `BytePairTokenizer`, or `SentencePieceTokenizer`.
+
+In this workflow we train a custom tokenizer on the IMDB data and design a backbone with
+custom transformer architecture. For simplicity we then train directly on the
+classification task. Interested in more details? We wrote an entire guide to pretraining
+and finetuning a custom transformer:
+https://keras.io/guides/keras_nlp/transformer_pretraining/
```

Comment:
I think this can be /guides/keras_nlp/transformer_pretraining/. Also consider not using a bare link